src/bitcoin-tx.cpp

```diff
@@ -642,10 +642,12 @@ static int CommandLineRawTx(int argc, char* argv[])
                 throw runtime_error("invalid transaction encoding");
 
             startArg = 2;
-        } else
+        } else {
+            txDecodeTmp = MakeTransactionRef();
```

Comment:
Whoops, I see. Initializing txDecodeTmp above (line 628) with a nullptr might make it more obvious to a casual reader that this points to nowhere initially (though you'd think the "Ref" part would've given it away...). No big deal though.